galileo_ros/src/galileo_legged_ros_implementation.cpp

```diff
@@ -271,7 +271,6 @@ void GalileoLeggedROSImplementation::CreateCost(casadi::Function &L, casadi::Fun
 
     casadi::SX X_ref = casadi::SX(X0);
     X_ref(casadi::Slice(states_->nh + states_->ndh, states_->nh + states_->ndh + 3)) = target_pos;
```

Comment:
This should probably use q_index

Reply:
I agree, but I don't think that's been merged yet. should we merge these branches, then rebase on top of them?